website/util/akismet.py

```diff
@@ -66,11 +69,15 @@ def check_comment(self, user_ip, user_agent, **kwargs):
         data['user_ip'] = user_ip
         data['user_agent'] = user_agent
 
-        res = requests.post(
-            '{}{}.{}/1.1/comment-check'.format(self.API_PROTOCOL, self.apikey, self.API_HOST),
-            data=data,
-            headers=self._default_headers
-        )
+        try:
+            res = requests.post(
```

Comment:
`requests.post` will not raise an exception for error responses unless you call `res.raise_for_status()`.